green/subprocess.py

```diff
@@ -225,7 +293,9 @@ def poolRunner(test_name, coverage_number=None, omit_patterns=[]):
         t.class_name  = 'N/A'
         t.description = 'Green encountered an error loading the unit test.'
         t.method_name = 'poolRunner'
+        result.startTest(t)
         result.addError(t, err)
+        result.stopTest(t)
```

Comment:
We "start" a fake test here to indicate to the parent that an error happened outside the test suite.